components/script/dom/focusevent.rs

```diff
@@ -47,6 +47,7 @@ impl FocusEvent {
         reflect_dom_object_with_proto(Box::new(FocusEvent::new_inherited()), window, proto, can_gc)
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
In future, can you keep clippy fixes like this to separate PRs?

Reply:
Alright, noted!